samples/subsys/fs/fs_sample/boards/ek_ra8d1.overlay

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&sdhc1 {
+	sdmmc {
```

Comment:
These same overlays appear all over for each test. Can we just define the SDMMC node within the base board devicetree?

Reply:
Yes, I added it into board devicetree and remove overlay for tests also. Thank you